moveit_core/trajectory_processing/src/ruckig_traj_smoothing.cpp

```diff
@@ -69,25 +67,29 @@ bool RuckigSmoothing::applySmoothing(robot_trajectory::RobotTrajectory& trajecto
   const size_t num_waypoints = trajectory.getWayPointCount();
```

Comment:
Probably not a big deal - but I noticed you started using right const in your updates. We still have left consts from previously; can we be consistent with the placement of const throughout the file?

Reply:
Good point, I'll make all the things right-const. It really is a little easier to read, I think